avoidance/QuadCopterStopAvoidance.hh

```diff
@@ -26,8 +26,12 @@ class QuadCopterStopAvoidance
     : public CollisionAvoidanceStrategy<MavQuadCopter>
 {
   public:
-    QuadCopterStopAvoidance(std::shared_ptr<MavQuadCopter> quadcopter);
+    QuadCopterStopAvoidance(std::shared_ptr<MavQuadCopter> quadcopter,
+                            double trigger_distance = DBL_MAX);
```

Comment:
I think we could have a better default here. Maybe set it to 2.5m like you did in your last patch? (Or maybe a bit bigger to be safe, but still a useful value).